pkg/runtime/cli.go

```diff
@@ -92,14 +92,15 @@ func FromFlags() (*DaprRuntime, error) {
 	metricsExporter.Options().AttachCmdFlags(flag.StringVar, flag.BoolVar)
 
 	// Finally parse the CLI flags!
-	flag.Parse()
+	// Ignore errors; CommandLine is set for ExitOnError.
```

Comment:
In which case, cmdline raise an error but do not affect?

Reply:
This is exactly the code and comment under the `flag.Parse()` function.  The comment is to show that if any error occurs, the code inside `CommandLine` will call exit so we do not need to handle errors here.